<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMakeProjectManager::Internal::ToolsSettingsWidget</class>
 <widget class="QWidget" name="XMakeProjectManager::Internal::ToolsSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>349</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontal_layout">
   <item>
    <layout class="QVBoxLayout" name="vertical_layout">
     <item>
      <widget class="QTreeView" name="xmake_list"/>
     </item>
     <item>
      <widget class="Utils::DetailsWidget" name="xmake_details" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vertical_layout_2">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clone_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="make_default_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Set as the default Meson executable to use when creating a new kit or when no value is set.</string>
       </property>
       <property name="text">
        <string>Make Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
